krabs/krabs/parse_types.hpp

```diff
@@ -253,6 +253,42 @@ namespace krabs {
     private:
     };
 
+    /**
+    * <summary>
+    * Used to handle parsing of POINTER types
+    */
+    struct pointer {
+        // Make the reference const and call it wild,
+        // As any attempt to derefernce or alter what it points to
+        // would be very undefined behaviour, as it
+        // is unlikley it is pointing to inside our process' memory
+        const void* wild_reference;
+
+        static pointer from_bytes(const BYTE* bytes, size_t size_in_bytes)
+        {
+            pointer pt;
+
+            // If 32-Bit, first parse as a uint32
+            // Then we can 'cast' that as pointer
+            if (size_in_bytes == sizeof(uint32_t)) {
+                uint64_t tmp = *(uint32_t*)bytes;
```

Comment:
I think we prefer static_casts in this code. Just be consistent with the rest of the file